pkg/issuer/acme/dns/cloudflare/cloudflare.go

```diff
@@ -62,6 +63,23 @@ func (c *DNSProvider) Present(domain, token, keyAuth string) error {
 		return err
 	}
 
+	record, err := c.findTxtRecord(fqdn)
+	if err != nil && err != errNoExistingRecord {
+		// this is a real error
+		return err
+	}
+	if record != nil {
+		if record.Content == value {
+			// the record is already set to the desired value
+			return nil
+		}
+
+		_, err = c.makeRequest("DELETE", fmt.Sprintf("/zones/%s/dns_records/%s", record.ZoneID, record.ID), nil)
```

Comment:
CloudFlare API support PUT, which updates record: https://api.cloudflare.com/#dns-records-for-a-zone-update-dns-record